package.json

```diff
@@ -3,46 +3,39 @@
   "description": "A grunt plugin to replace url from static files such as img,js,css an put inline in a template",
   "version": "0.1.6",
   "homepage": "https://github.com/dayvson/grunt-static-inline",
-  "author": {
-    "name": "Maxwell Da Silva",
-    "email": "dayvson@gmail.com"
-  },
+  "author": "Maxwell Da Silva <dayvson@gmail.com>",
+  "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/dayvson/grunt-static-inline"
+    "url": "git+https://github.com/dayvson/grunt-static-inline.git"
   },
   "bugs": {
     "url": "https://github.com/dayvson/grunt-static-inline/issues"
   },
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://github.com/dayvson/grunt-static-inline/blob/master/LICENSE"
-    }
-  ],
-  "main": "Gruntfile.js",
   "engines": {
-    "node": ">= 0.8.0"
+    "node": ">=0.10.0"
   },
-  "preferGlobal": true,
-
   "scripts": {
-    "test": "node node_modules/grunt-cli/bin/grunt test"
+    "test": "grunt test"
```

Comment:
this assumes that you have grunt install globally which I don't think is a good idea.